framework/tests/all-systems/AnoymousAndInnerClass.java

```diff
@@ -5,6 +5,10 @@ public MyInnerClass() {}
         public MyInnerClass(String s) {}
 
         public MyInnerClass(int... i) {}
+
+        public MyInnerClass(AnoymousAndInnerClass c) {}
```

Comment:
Typo: `Anonymous`. Can you refactor the class name and the file name?

Reply:
Thanks